codespell_lib/data/dictionary.txt

```diff
@@ -29012,6 +29041,8 @@ speciic->specific
 speciied->specified
 speciifc->specific
 speciifed->specified
+specilization->specialization
+specilizations->specializations
```

Comment:
Can you separately add the `isa` versions for BE.

Reply:
Added in 6af8b7f